src/graphql/types/object/user.ts

```diff
@@ -15,15 +15,17 @@ import { UnknownClientError } from "@core/error"
 
 const mainUserFields = () => ({
   id: { type: GT.NonNullID },
-  phone: { type: GT.NonNull(Phone) },
-  username: { type: Username },
+  phone: { type: GT.NonNull(Phone), description: "Phone number with international calling code." },
+  username: { type: Username, description: "Optional immutable user friendly identifier." },
   language: {
     type: GT.NonNull(Language),
     resolve: (source) => source.language,
+    description: "Preferred language for user.\nWhen value is 'default' the intent is to use preferred language from OS settings."
```

Comment:
Can we use multiline string instead of `\n`?

Reply:
I have switched out all strings which use new lines to use multi line strings.